src/app/shared/services/translation.service.ts

```diff
@@ -42,8 +42,9 @@ constructor(
             this.dictionary.next(this.storage.retrieve('lang'));
             this.setIndex();
         } else {
-            this.dictionary.next(HindiLang);
-            this.storage.store('lang', HindiLang);
+            const lang = HindiLang.sort(this.compare_text);
```

Comment:
@Nishanthh The json file is already sorted in alphabetical order, right? I run a ruby script that sorts it before I upload it.

Reply:
@avikt Yes bro, since i added the missing text in JSON also need to be sorted.

Reply:
@Nishanthh Ok. I can run the script again and make it alphabetically sorted again though